migration/iD11/oracle/202303171334_IDEMPIERE-5567.sql

```diff
@@ -0,0 +1,250 @@
+;
+
+-- Mar 17, 2023, 1:34:45 PM CET
+INSERT INTO AD_Column (AD_Column_ID,Version,Name,Description,Help,AD_Table_ID,ColumnName,FieldLength,IsKey,IsParent,IsMandatory,IsTranslated,IsIdentifier,IsEncrypted,AD_Reference_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,AD_Element_ID,IsUpdateable,IsSelectionColumn,EntityType,IsAlwaysUpdateable,AD_Column_UU,IsToolbarButton) VALUES (215789,0.0,'Comment/Help','Comment or Hint','The Help field contains a hint, comment or help about the use of this item.',200383,'Help',2000,'N','N','N','N','N','N',10,0,0,'Y',TO_TIMESTAMP('2023-03-17 13:34:45','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2023-03-17 13:34:45','YYYY-MM-DD HH24:MI:SS'),100,326,'Y','N','D','N','d9291e97-85da-4817-816c-7efc9971dcce','N')
+;
+
+-- Mar 17, 2023, 1:35:40 PM CET
+UPDATE AD_Column SET IsAllowCopy='N', FKConstraintName='ADClient_TestUU', FKConstraintType='N',Updated=TO_TIMESTAMP('2023-03-17 13:35:40','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=215778
+;
+
+-- Mar 17, 2023, 1:35:40 PM CET
+UPDATE AD_Column SET IsAllowCopy='N', FKConstraintName='ADOrg_TestUU', FKConstraintType='N',Updated=TO_TIMESTAMP('2023-03-17 13:35:40','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=215779
+;
+
+-- Mar 17, 2023, 1:35:40 PM CET
+UPDATE AD_Column SET IsAllowCopy='N', FKConstraintName='CreatedBy_TestUU', FKConstraintType='N',Updated=TO_TIMESTAMP('2023-03-17 13:35:40','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=215781
+;
+
+-- Mar 17, 2023, 1:35:40 PM CET
+UPDATE AD_Column SET IsAllowCopy='N', FKConstraintName='UpdatedBy_TestUU', FKConstraintType='N',Updated=TO_TIMESTAMP('2023-03-17 13:35:40','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=215783
+;
+
+-- Mar 17, 2023, 1:35:40 PM CET
+CREATE TABLE TestUU (AD_Client_ID NUMBER(10) NOT NULL, AD_Org_ID NUMBER(10) NOT NULL, Created DATE NOT NULL, CreatedBy NUMBER(10) NOT NULL, Description VARCHAR2(255 CHAR) DEFAULT NULL , Help VARCHAR2(2000 CHAR) DEFAULT NULL , IsActive CHAR(1) DEFAULT 'Y' CHECK (IsActive IN ('Y','N')) NOT NULL, Name VARCHAR2(60 CHAR) NOT NULL, TestUU_UU VARCHAR2(36 CHAR) DEFAULT NULL , Updated DATE NOT NULL, UpdatedBy NUMBER(10) NOT NULL, Value VARCHAR2(40 CHAR) DEFAULT NULL , CONSTRAINT TestUU_UU_idx UNIQUE (TestUU_UU))
+;
```

Comment:
UUID table should be created with _UU column as the primary key constraint.

Reply:
> UUID table should be created with _UU column as the primary key constraint.

Ready @hengsin - implemented with commit: https://github.com/idempiere/idempiere/pull/1727/commits/16d531042dce3d350c8b74ea1ca0e86f9d76bd41